config/environments/development.rb

```diff
@@ -34,4 +34,16 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+      authentication: ENV['SMTP_SETTINGS_AUTHENTICATION'],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.